Packs/SymantecCloudSOC/Integrations/SymantecCloudSOC/SymantecCloudSOC.py

```diff
@@ -0,0 +1,489 @@
+                        last_run_ids.append(event_id)
+                # The event has a timestamp we have not yet fetched meaning it is a new event
+                else:
+                    new_events.append(event)
+                    new_events_ids.append(event_id)
+                    # If the event has a timestamp newer than the saved one, we will update the last run to the
+                    # current event time
+                    event_timestamp_date = datetime.strptime(event_timestamp, DATE_FORMAT_SYMANTEC)
+                    if new_last_run_time_date < event_timestamp_date:
+                        new_last_run_time = event_timestamp
+                    else:
+                        demisto.debug(f'SymantecEventCollector: new_last_run_time is {new_last_run_time}'
+                                      f'event_timestamp is {event_timestamp} and command is {demisto.command()}')
+                new_last_run_time_date = datetime.strptime(new_last_run_time, DATE_FORMAT_SYMANTEC)
+                add_fields_to_event(event, log_type)
+
+        # If we have received events with a newer time (new_event_ids list) we save them,
+        # otherwise we save the list that include the old ids together with the new event ids (last_run_ids).
+        last_run_time_date = datetime.strptime(last_run_time, DATE_FORMAT_SYMANTEC)
+        if (last_run_time_date < new_last_run_time_date) and new_events_ids:
+            new_last_run[f'{log_type}-ids'] = new_events_ids
+        else:
+            new_events_ids.extend(last_run_ids)
+            new_last_run[f'{log_type}-ids'] = new_events_ids
+        # new last_run is new_last_run_time if exists else last_run_time
+        new_last_run["last_run"] = new_last_run_time or last_run_time
+    # If we dont have any events last_run is still the last run time
+    elif last_run_time:
+        new_last_run["last_run"] = last_run_time
+    demisto.debug(f'SymantecEventCollector: Setting new last run - {new_last_run} for {log_type}'
+                  f' and the command is {demisto.command()}')
+    return new_events, new_last_run
+
+
+def get_all_events_for_log_type(client: Client, max_fetch: int, log_type: str, last_run: Dict[str, dict],
+                                first_fetch_time: str, first_fetch_time_investigate: str):
+    """
+    Gets all the events for a specific log type.
+    Args:
+        client (Client): Symantec Cloud SOC client to use.
+        max_fetch (int): Maximum numbers of events per fetch.
+        log_type (str): The log type.
+        last_run (Dict): last run object.
+        first_fetch_time (str): The timestamp on when to start fetching events.
+        first_fetch_time_investigate (str): The timestamp on when to start fetching events for investigate.
+    Returns:
+        Client: Client class to interact with Symantec Cloud SOC service API.
+    """
+    demisto.debug(f'SymantecEventCollector: last_run is {last_run} and the command is {demisto.command()}')
```

Comment:
```suggestion
    demisto.debug(f'SymantecEventCollector: last_run is {last_run}')
```
Remove from all logs - not necessary